ibm_mq/datadog_checks/ibm_mq/collectors/stats_collector.py

```diff
@@ -37,14 +37,14 @@ def collect(self, queue_manager):
         """
         self.log.debug("Collecting stats newer than %s", self.config.instance_creation_datetime)
         queue = Queue(queue_manager, STATISTICS_QUEUE_NAME)
-
+        pcf = None
         try:
             # It's expected for the loop to stop when pymqi.MQMIError is raised with reason MQRC_NO_MSG_AVAILABLE.
+            pcf = pymqi.PCFExecute()
```

Comment:
looks like there's a bug here: https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=93599&view=logs&j=910f82a4-622d-5ace-f877-8b75c045486c&t=12901f7b-620f-5414-43d1-3ae27e5163d7&l=1435

Reply:
Good catch, I will remove this change since I don't think `pymqi.PCFExecute()` is actually supposed to be initialized here.